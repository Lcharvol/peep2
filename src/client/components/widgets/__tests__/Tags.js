import React from 'react';
import 'jest-styled-components';
import { shallow } from 'enzyme'; // eslint-disable-line
import Tags from '../Tags';

describe('<Tags />', () => {
  test('should render a <Tags />', () => {
    const tree = shallow(<Tags />);
    expect(tree).toMatchSnapshot();
  });
  test('should <Tags /> have display flex', () => {
    const tree = shallow(<Tags />);
    expect(tree).toHaveStyleRule('display', 'flex');
  });
});
