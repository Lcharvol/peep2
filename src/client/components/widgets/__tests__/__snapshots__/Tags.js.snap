// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Tags /> should render a <Tags /> 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <div
    className="sc-bdVaJa ifKqTt"
/>,
  "nodes": Array [
    <div
      className="sc-bdVaJa ifKqTt"
/>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_context": Object {},
    "_element": <styled.div />,
    "_instance": StyledComponent {
      "attrs": Object {},
      "context": Object {},
      "props": Object {},
      "refs": Object {},
      "state": Object {
        "generatedClassName": "ifKqTt",
        "theme": null,
      },
      "unsubscribeId": -1,
      "updater": Updater {
        "_renderer": [Circular],
      },
    },
    "_newState": Object {
      "generatedClassName": "ifKqTt",
      "theme": null,
    },
    "_rendered": <div
      className="sc-bdVaJa ifKqTt"
/>,
    "_rendering": false,
    "_updater": Updater {
      "_renderer": [Circular],
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <styled.div />,
}
`;
